lib/engine/game/g_18_ms.rb

```diff
@@ -62,12 +62,22 @@ def new_operating_round(round_num = 1)
           @phase.next! if @turn == 1 && round_num == 2
         end
 
+        if round_num == 1
+          @players.each do |p|
+            next unless p.cash.negative?
+
+            debt = -p.cash
+            interest = (debt / 2 + 0.99).floor
```

Comment:
what's the point of the 0.99?

int / int = int

Reply:
My idea was that
- if debt is 50, interest should be 25
- if debt is 51, interest should be 26

So (51 / 2.0 + 0.99) = 26.49 which floor gives 26
And (50 / 2.0 + 0.99) = 25.99 which floor gives 25

This is a VERY minor thing. 

But in case of debt is 1, 1 / 2 = 0.
Which would mean
  Player 1 has to borrow another $0 as being in debt at the end of SR

Reply:
But I have changed to + 0.5; which might make it clearer that it is just to get correct rounding.

The 0.99 was just to ensure that any problem with float handling would not cause any problem.